backend/apps/images/main.py

```diff
@@ -68,14 +68,20 @@
 
 app.state.MODEL = IMAGE_GENERATION_MODEL
 
-
 app.state.AUTOMATIC1111_BASE_URL = AUTOMATIC1111_BASE_URL
 app.state.COMFYUI_BASE_URL = COMFYUI_BASE_URL
 
-
 app.state.IMAGE_SIZE = IMAGE_SIZE
 app.state.IMAGE_STEPS = IMAGE_STEPS
 
+app.state.AUTOMATIC1111_API_AUTH = os.getenv("AUTOMATIC1111_API_AUTH")
```

Comment:
This should be imported via the config block in line 34 with the other variables, `import os` is thus not necessary either.